### PR TITLE
pages*: fix brand and technical names

### PR DESCRIPTION
**Pull request by vitorhcl**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

I kept some of them because they are already covered by #4362 (git-archive), #12107 (java) and #12132 (cs-java):
```sh
$ grep -RE '^[>-].+ java |^[>-].+ zip ' *
common/git-archive.md:- Create a zip archive from the current HEAD and print it to `stdout`:
common/git-archive.md:- Same as above, but write the zip archive to file:
common/java.md:- Display usage information for the java command:
common/cs-java.md:> The `java` and `java-home` commands fetch and install JVMs. The java command runs them too.
```

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages/common/bzfgrep.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # bzfgrep
 
-> Find any fixed strings separated by new lines in bzip2 compressed files using fgrep.
+> Find any fixed strings separated by new lines in `bzip2` compressed files using fgrep.
```
Comment (acuteenvy) [inline pages/common/bzfgrep.md:3]:
```suggestion
> Find any fixed strings separated by new lines in `bzip2` compressed files using `fgrep`.
```

File: pages/common/bzgrep.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # bzgrep
 
-> Find patterns in bzip2 compressed files using grep.
+> Find patterns in `bzip2` compressed files using grep.
```
Comment (acuteenvy) [inline pages/common/bzgrep.md:3]:
```suggestion
> Find patterns in `bzip2` compressed files using `grep`.
```

File: pages/common/cs-java.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 # cs java
 
-> The java and java-home commands fetch and install JVMs. The java command runs them too.
+> The `java` and `java-home` commands fetch and install JVMs. The java command runs them too.
```
Comment (acuteenvy) [inline pages/common/cs-java.md:3]:
```suggestion
> The `java` and `java-home` commands fetch and install JVMs. The `java` command runs them too.
```

File: pages/common/cs-launch.md [hunk 4]
```diff
@@ -15,6 +15,6 @@
 
 `cs launch {{group_id}}:{{artifact_id}}:{{artifact_version}} --main-class {{path/to/main_class_file}}`
 
-- Launch an application with specific java options and a jvm memory ones:
+- Launch an application with specific Java options and a jvm memory ones:
```
Comment (acuteenvy) [inline pages/common/cs-launch.md:18]:
```suggestion
- Launch an application with specific Java options and JVM memory ones:
```

File: pages/common/gunzip.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # gunzip
 
-> Extract file(s) from a gzip (.gz) archive.
+> Extract file(s) from a `gzip` (.gz) archive.
```
Comment (acuteenvy) [inline pages/common/gunzip.md:3]:
```suggestion
> Extract file(s) from a `gzip` (`.gz`) archive.
```

File: pages/common/rarcrack.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # rarcrack
 
-> Password cracker for rar, zip and 7z archives.
+> Password cracker for `rar`, `zip` and `7zip` archives.
```
Comment (acuteenvy) [inline pages/common/rarcrack.md:3]: The extension for 7zip archives is `7z`, and so is the command.

```suggestion
> Password cracker for `rar`, `zip` and `7z` archives.
```

File: pages/linux/atool.md [hunk 7]
```diff
@@ -3,18 +3,18 @@
...
+- List files in a `zip` archive:
 
 `atool --list {{path/to/archive.zip}}`
 
 - Unpack a tar.gz archive into a new subdirectory (or current directory if it contains only one file):
 
 `atool --extract {{path/to/archive.tar.gz}}`
 
-- Create a new 7zip archive with two files:
+- Create a new `7zip` archive with two files:
```
Comment (acuteenvy) [inline pages/linux/atool.md:14]:
```suggestion
- Create a new `7z` archive with two files:
```

File: pages/windows/select-string.md [hunk 8]
```diff
@@ -2,7 +2,7 @@
 
 > Finds text in strings and files in PowerShell.
 > This command can only be used through PowerShell.
-> You can use `Select-String` similar to grep in UNIX or findstr.exe in Windows.
+> You can use `Select-String` similar to `grep` in UNIX or findstr.exe in Windows.
```
Comment (acuteenvy) [inline pages/windows/select-string.md:5]:
```suggestion
> You can use `Select-String` similar to `grep` in UNIX or `findstr.exe` in Windows.
```

**Review by acuteenvy — COMMENTED** [review]
Git is written with a capital letter in most pages. Do we want to change that now?

**Comment by tricantivu** [comment]
Should protocol (e.g. WPA) and file format names (e.g. ELF) be considered technical terms? I think I see a few of these in the pages for `aircrack-ng`

**Comment by vitorhcl** [comment]
> Git is written with a capital letter in most pages. Do we want to change that now?

Fair enough, where it is spelled as "git" we can use "Git".

**Review by kbdharun — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**